recipes/rocksdb/all/conanfile.py

```diff
@@ -22,7 +23,10 @@ class RocksDB(ConanFile):
         "with_lz4": [True, False],
         "with_zlib": [True, False],
         "with_zstd": [True, False],
-        "with_tbb": [True, False]
+        "with_tbb": [True, False],
+        "with_jemalloc": [True, False],
+        "enable_sse": [False, "sse42", "avx2"],
```

Comment:
this option make sense only on `x86` and `x86_64` and should be removed otherwise

Reply:
Done